packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerHeader/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { memo } from 'react';
+import { Image, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { Avatar, styled, H3 } from '@apollosproject/ui-kit';
+
+const HeaderContent = styled(() => ({
+  alignItems: 'center',
+}))(View);
+
+const Title = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 0.5,
+  marginBottom: theme.sizing.baseUnit * 2,
+}))(H3);
+
+const AddPrayerHeader = memo(({ imgSrc, title }) => (
+  <HeaderContent>
+    <Avatar source={imgSrc} />
```

Comment:
the `<Avatar/>` component defaults to `size={'small'}`. But that doesn't seem to be the size we need. Try `size={'medium or large'}`

Reply:
👍